exercise03/README.md

```diff
@@ -0,0 +1,3 @@
+## Exercise 3
+
+Given the tables and records defined `setup.sql`.  Write a query that returns all records and a count where `name` is not unique.
```

Comment:
Of course, just let me know which count you had in mind. :) Or, actually, maybe we could ask for an aggregation (using ARRAY_AGG in postgres) and count of records for each duplicate?